contrib/spice-gtk3/patches/0004.patch

```diff
@@ -0,0 +1,164 @@
+From 7e87382b585a799ced543b0c3fd127b2a813285f Mon Sep 17 00:00:00 2001
```

Comment:
afaict these two huge patches are just to work around undef symbols, better to pass -Wl,--undefined-version instead since there's so many